src/main/java/org/candlepin/pki/impl/JSSProviderLoader.java

```diff
@@ -83,8 +94,15 @@ public static void addLibraryPath(String pathToAdd)  {
         }
     }
 
-    private JSSProviderLoader() {
-        // static methods only
+    /**
+     * Fetches a string representing the JSS version loaded at runtime. This function should never
+     * return null.
+     *
+     * @return
+     *  JSS version string
+     */
+    public static String getJSSVersion() {
+        return JSSProvider.class.getPackage().getSpecificationVersion();
```

Comment:
Do we need to be concerned about a possible NPE if the JSSProvider is ever missing from the classpath? I'm wondering if it would be worth checking if getPackage() returns a null and throwing an exception if it is null.

Reply:
Nope. That's a compile-time check, and if the class is missing, we'll get ClassNotFoundExceptions as the JSSProviderLoader is loaded.

The way we load and use JSS is weird, and, honestly, if I had been available at the time this decision was made, I would have pushed hard to not get into this type of situation.

Reply:
+1